application/backend/src/main/java/ai/typerefinery/websight/models/components/forms/Button.java

```diff
@@ -168,17 +185,15 @@ protected void init() {
 
         if (grid != null && style != null) {
             String buttonCls = "btn";
-            if (StringUtils.isNotBlank(buttonVariant)) {
+            if (StringUtils.isNotBlank(buttonStyle)) {
```

Comment:
Also noticed one thing if buttonStyle is blank add the default as primary.

Reply:
added.